docs/src/equations.md

```diff
@@ -65,13 +65,16 @@ We make use of the following operators
 * ``\bar{\boldsymbol{u}} = \boldsymbol{u}_h + I_{c}(\boldsymbol{u}_v)``
 
 * ``\Phi = g z`` is the geopotential, where ``g`` is the gravitational acceleration rate and ``z`` is altitude above the mean sea level.
+* ``\boldsymbol{b} = - (\rho - \rho_h) / \rho \nabla \Phi`` is the reduced gravitational accleration
+* ``\rho_h`` is the reference state density
 * ``K = \tfrac{1}{2} \|\boldsymbol{u}\|^2 `` is the specific kinetic energy (J/kg), reconstructed at cell centers by
   ```math
   K = \tfrac{1}{2} (\boldsymbol{u}_{h} \cdot \boldsymbol{u}_{h} + 2 \boldsymbol{u}_{h} \cdot \boldsymbol{u}_{v} + I_{c}(\boldsymbol{u}_{v} \cdot \boldsymbol{u}_{v})),
   ```
   where ``\boldsymbol{u}_{h}`` is defined on cell-centers, ``\boldsymbol{u}_{v}`` is defined on cell-faces, and ``I_{c}`` is an interpolation operation using covariant components.  
 
 * ``p`` is air pressure, derived from the thermodynamic state, reconstructed at cell centers.
+* ``p_h`` is the reference state pressure. It is related to the reference state density by hydrostatic balance: ``\Delta p_h = - \rho_h \nabla \Phi``.
```

Comment:
what are `\Delta` and `\nabla` here?

Reply:
Ah, they should be both \nabla. And it is the gradient.

Reply:
And here p_h and \rho_h are actually in analytical balance. I will add a note on this.